GiTinder/Models/GitHubProfile.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GiTinder.Models
```

Comment:
@Jonathan-Bonnin this should be in the GithubResponses namespace and we should have only one